pkg/nuctl/command/deploy.go

```diff
@@ -888,6 +888,7 @@ func (d *deployCommandeer) resolveRequestHeaders() map[string]string {
 		// add a header that will tell the API to only deploy imported functions
 		requestHeaders[headers.ImportedFunctionOnly] = "true"
 	}
+	requestHeaders[headers.VerifyExternalRegistry] = "true"
```

Comment:
Why is this always true and not configurable with a flag?
We would like to allow redeploying with nuctl even if the registry is internal (in non-upgrade flow or non-iguazio users for example)